docs/processes/release/tasks/docs.md

```diff
@@ -374,21 +374,19 @@ Note: This is also referenced below [Upload API documentations](docs.md#upload-a
 Build the API documentation pages for the new release:
 
 ```
-ant releaseDocs
+release_tools/github_release.sh
 ```
+The API documentation will be found in targets/github_release/
 
-#### Alternative method:
+Next, update the documents for this version:
+* Copy the fulljavadoc.jar to the to icu-docs/apidoc directory, either dev (for release candidates) or released (for public distribution).
```

Comment:
Typo?
```suggestion
* Copy the full javadoc.jar to the to icu-docs/apidoc directory, either dev (for release candidates) or released (for public distribution).
```

Reply:
No, the file is actually called "icu4j-75_1-fulljavadoc.jar". There's another file "icu4j-75_1-javadoc.jar" which is not the one to use.

Reply:
No, there are 2 javadoc files, icu4j-75_1-javadoc.jar and icu4j-75_1-fulljavadoc.jar. The "full" one is what should be used.

Reply:
Might be best to document it as `icu4j-<version>-fulljavadoc.jar` then.
I think it would avoid confusion.

---

In the longer run I think that creating jar + unzipped folder should be done by the script.
No reason to do it by hand.